__tests__/app.test.js

```diff
@@ -26,3 +27,23 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: Responds with an object of all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.result).toEqual(data.topicData)
+      })
+  })
+})
+describe("Not found error", () =>{
+  test('404: responds with not found', async () => {
+    return request(app)
+    .get("/api/topi")
+    .expect(404)
+    .then((response) =>{
+      expect(response.text).toBe("{\"msg\":\"Route not found\"}")
```

Comment:
Something has gone wrong with the formatting of your message here. Remember that you should be sending back an object with a key of msg and a value of the error message.